README.md

```diff
@@ -1,90 +1,85 @@
-# Ember Data Filter
+# [DEPRECATED] Ember Data Filter
 
 ## Install
 
 ### Ember CLI
 
 `ember install ember-data-filter`
 
-### Rails and Globals Mode apps
-
-You'll need to load a custom script that enables the script **before you
-load Ember**. For example, if you call your file config.js:
+### Configuration
 
 ```js
-// /config.js
+//  app/services/store.js
+import Store from 'ember-data/store';
+import FilterMixin from 'ember-data-filter/mixins/filter';
 
-window.EmberENV = window.EmberENV || {};
-window.EmberENV.ENABLE_DS_FILTER = true;
+export default Store.extend(FilterMixin);
```

Comment:
Maybe mention something about how this plays with common addons which extend the store?

Reply:
👍